docs/cudf/source/Working-with-missing-data.ipynb

```diff
@@ -0,0 +1,3032 @@
+{
```

Comment:
<p>Also mention about <code>dropna </code> option if they want to preserve it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cudf/pull/7010/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>

Reply:
<p>Sure, Added this one :+1:</p><p> </p>